### PR TITLE
compile less files in Dockerfile

### PR DESCRIPTION
**Pull request by nataly87s**

### CODE REVIEW

File: services/editor/Dockerfile [hunk 1]
```diff
@@ -15,6 +18,7 @@ RUN yarn
 
 COPY . /src/
 RUN yarn test
+RUN yarn run compile-less
```
Comment (Yshayy) [inline services/editor/Dockerfile]: I think build should activate compile less

Comment (Yshayy) [inline services/editor/Dockerfile]: why global? can it be just a dev-dependency (like babel)

Comment (nataly87s) [inline services/editor/Dockerfile]: I thought about it, the problem is that `compile-less` is a bash script, so if you run `build` in windows it will fail

**Comment by Yshayy** [comment]
We need to find a way to run compile-less on both windows and mac and run it on dev (start-full-dev) environments as well.
How much time does it take?

**Comment by nataly87s** [comment]
I think it took 21 seconds the last time i ran it, but i'll try to find a better way, see what less has to offer

**Comment by nataly87s** [comment]
will add less loader to webpack after ejecting create-react-app

**Pull request closed without merging**